.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Testing the electron app 
```

Comment:
Rename it to "Testing the frontend"

Reply:
Its not electron app specific.

The frontend is used by electron app and react app as well